ecdsa.go

```diff
@@ -1,22 +1,30 @@
 package cose
 
 import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
 	"errors"
+	"io"
 	"math/big"
+
+	"golang.org/x/crypto/cryptobyte"
+	"golang.org/x/crypto/cryptobyte/asn1"

```

Comment:
`golang.org/x/crypto/cryptobyte` does not appear in the `go.mod` file. Run `go mod tidy` to fix it.

Reply:
I will fix all the `go.mod` when I finish all the coding. I think I just have `Verifier` and `COSE_Sign` left.